helm-v3/006-repositories.md

```diff
@@ -2,10 +2,22 @@
 
 The following changes will be made to the repository subsystem:
 
+- A new `helm login` command will be added for authenticating against a repository
+- A new `helm push` command will be added for pushing charts to a repository
 - The `helm serve` command will be removed
-- A new `helm push` verb will be added for pushing charts to a repository
+- [ChartMuseum](https://github.com/helm/chartmuseum) will be the Helm subproject
+  dedicated to hosting your own repository
+- [Monocular](https://github.com/helm/monocular) will be the Helm subproject
+  dedicated to distributed repo discovery, and the software powering
+  [Hub](https://github.com/helm/hub)
 
-> This document is in-progress and needs input from Chart Museum.
+## Authenticating Against A Repository
+
+*In progress. Please see [chartmuseum#59](https://github.com/helm/chartmuseum/issues/59).*
```

Comment:
Can we pull in an overview of the authentication model here? The parts that are relevant to the Helm interaction, anyway.

It might be time to create a specs repo. I'll kickoff that process.